examples/telehealth-tutorial-app/src/App.css

```diff
@@ -0,0 +1,198 @@
+.App {
+  text-align: center;
+  height: 100vh;
+  width: 100%;
+  display: flex;
+}
+
+.left-section {
```

Comment:
might be nice to have a z index here
![image](https://github.com/whereby/browser-sdk/assets/138459303/92156389-c0a3-43cb-a5b3-579bc61e51f0)

Reply:
good shout, will add